docs/guides/building/macos.md

```diff
@@ -0,0 +1,72 @@
+:::
+
+## Binary Targets
+
+macOS applications can target Apple Silicon, Intel-based Mac computers, or Universal macOS binaries that work on both architectures. By default, the Tauri CLI uses your machine's architecture, but you can configure a different target using the `--target` flag:
+
+```shell
+tauri build --target aarch64-apple-darwin
+```
+
+Supported targets are:
+
+- `aarch64-apple-darwin`: Apple Silicon (using the M1 processor architecture)
+- `x86_64-apple-darwin`: Intel-based Macs
+- `universal-apple-darwin`: produces a [Universal macOS Binary] that runs on both Apple silicon and Intel-based Macs.
+
+While Apple silicon machines can run applications compiled for Intel-based Macs through a translation layer called [Rosetta], this leads to a reduction in performance due to processor instruction translations. It is common practice to let the user choose the correct target when downloading the app, but you can also choose to distribute a [Universal Binary][universal macos binary]. Universal Binaries include both `aarch64` and `x86_64` executables, giving you the best experience on both architectures. Note, however, that this increases your bundle size significantly.
+
+## Application Bundle Customization
+
+The Tauri configuration file provides the following options to customize your application bundle:
+
+- **Bundle name:** Your app's human-readable name. Configured by the [`package.productName`] property.
+- **Bundle version:** Your app's version. Configured by the [`package.version`] property.
+- **Application category:** The category that describes your app. Configured by the [`tauri.bundle.category`] property. You can see a list of macOS categories [here][macos app categories].
+- **Copyright:** A copyright string associated with your app. Configured by the [`tauri.bundle.copyright`] property.
+- **Bundle icon:** Your app's icon. Uses the first `.icns` file listed in the [`tauri.bundle.icon`] array.
+- **Minimum system version:** Configured by the [`tauri.bundle.macOS.minimumSystemVersion`] property.
+- **DMG license file:** A license that is added to the `.dmg` file. Configure by the [`tauri.bundle.macOS.license`] property.
+- **[Entitlements.plist file]:** Entitlements control what APIs your app will have access to. Configured by the [`tauri.bundle.macOS.entitlements`] property.
+- **Exception domain:** an insecure domain that your application can access such as a `localhost` or a remote `http` domain. It is a convenience configuration around `NSAppTransportSecurity > NSExceptionDomains` setting `NSExceptionAllowsInsecureHTTPLoads` and `NSIncludesSubdomains` to true. See [`tauri.bundle.macOS.exceptionDomain`] for more information.
+
+:::info
+These options generate the application bundle [Info.plist file]. You can extend the generated file with your own `Info.plist` file stored on the Tauri folder (`src-tauri` by default). The CLI merges both `.plist` files on production, and the core layer embeds it on the binary on development.
```

Comment:
is it just me or should it be _in_ prod, _in_ the binary, _in_ devel ?

Reply:
How about now?